lib/host-sp-messages/src/lib.rs

```diff
@@ -515,6 +515,18 @@ pub enum InventoryData {
         capacity: u8,
         unique_id: [u8; 8],
     },
+
+    /// Cosmo host flash
+    W25q01jvzeiq {
+        mfr_id: u8,
+        memory_type: u8,
+        capacity: u8,
+
+        /// 64-bit unique ID for die 0
+        ///
+        /// Note that each 512M-bit die has a separate ID!
+        unique_id: [u8; 8],
```

Comment:
I would probably include both die's `unique_id` values. Given that this consists of two dies, I would probably model it that way in topology. It's hard to say how practical it is to have a single die failure, but given that we are choosing which die we're addressing and the issues there, that gives us something.

Reviewing what we have from them, it's not clear if we'd actually ever get failure/recall information that is die specific from the manufacturer or if we'd ultimately just get date code information. It's probably just the latter to be honest, which we have no means of electronically identifying it appears.